tutorials/plot_background_filtering.py

```diff
@@ -483,15 +524,232 @@ def plot_signal(x, offset):
 plt.show()
 
 ###############################################################################
-# Filtering in MNE-Python
-# =======================
+# Some pitfalls of filtering
+# ==========================
+# Multiple recent papers have noted potential risks of drawing
+# errant inferences due to misapplication of filters.
+#
+# Low-pass problems
+# -----------------
+# Filters in general, especially those that are acausal (zero-phase), can make
+# activity appear to occur earlier or later than it truly did. As
+# mentioned in VanRullen 2011 [3]_, investigations of commonly (at the time)
+# used low-pass filters created artifacts. However, such deleterious effects
```

Comment:
Oh, I didn't mean this would belong here, just that I would guess that what VanRullen reported there depends also on the simulated data., not just the filter.